test/Integration/Dialect/TOSA/i8xi16_sub_elem/dut.cc

```diff
@@ -0,0 +1,20 @@
+void dut(int8_t * restrict v1, int16_t * restrict v2, int32_t * restrict v3) {
```

Comment:
It seems that you haven't uploaded testbench.cc and defines.h file for this test.

Reply:
Similar to the bf16xbf16_sub_elem_2d_broadcast_2d test. This is to fix a XFAIL test, so only adding the dut.cc for reference.